util/set/string_set.go

```diff
@@ -31,3 +31,12 @@ func (s StringSet) Exist(val string) bool {
 func (s StringSet) Insert(val string) {
 	s[val] = struct{}{}
 }
+
+// InsertIfNotExist inserts `val` into `s` if it not exists in `s`.
+func (s StringSet) InsertIfNotExist(val string) bool {
```

Comment:
is it faster than `Insert()`?

Reply:
I think not.
This function is just introduced for the cases that we need to check `Exists` first.

Reply:
So I think we can replace this function with `Insert()`.